common/src/main/java/com/wynntils/features/ui/WynncraftButtonFeature.java

```diff
@@ -66,6 +76,21 @@ private void addWynncraftButton(TitleScreen titleScreen) {
         titleScreen.addRenderableWidget(wynncraftButton);
     }
 
+    /** Connects to the server on "startup".
+     *  Requires the WynncraftButtonFeature to be enabled.
+     */
```

Comment:
There is no reason to have this comment. More is not always better, and that is the case here.  All the info you describe here is true but can be collected from the feature's class itself.

Reply:
removed.
it was a wip comment anyway but the second part the "Requires the WynncraftButtonFeature to be enabled." 
Ive actually spent easily an hour or two to find out.